components/com_users/tmpl/method/edit.php

```diff
@@ -0,0 +1,164 @@
+		<input type="hidden" name="<?php echo $this->escape($key) ?>" value="<?php echo $this->escape($value) ?>">
+		<?php endforeach; ?>
+		<?php endif; ?>
+
+		<?php if (!empty($this->title)): ?>
+		<?php if (!empty($this->renderOptions['help_url'])): ?>
+		<span class="float-end">
+			<a href="<?php echo $this->renderOptions['help_url'] ?>"
+			   class="btn btn-sm btn-default btn-inverse btn-dark"
+			   target="_blank"
+			>
+				<span class="icon icon-question-sign" aria-hidden="true"></span>
+				<span class="visually-hidden"><?php echo Text::_('JHELP') ?></span>
+			</a>
+		</span>
+		<?php endif;?>
+		<h3 id="com-users-method-edit-head">
+			<?php echo Text::_($this->title) ?>
+		</h3>
+		<?php endif; ?>
+
+		<div class="row mb-3">
+			<label class="col-sm-3 col-form-label hasTooltip"
+				for="com-users-method-edit-title"
+				title="<?php echo $this->escape(Text::_('COM_USERS_MFA_EDIT_FIELD_TITLE_DESC')) ?>">
+				<?php echo Text::_('COM_USERS_MFA_EDIT_FIELD_TITLE'); ?>
+			</label>
+			<div class="col-sm-9">
+				<input type="text"
+						class="form-control"
+						id="com-users-method-edit-title"
+						name="title"
+						value="<?php echo $this->escape($this->record->title) ?>"
+						placeholder="<?php echo Text::_('COM_USERS_MFA_EDIT_FIELD_TITLE_DESC') ?>">
+			</div>
+		</div>
+
+		<div class="row mb-3">
+			<div class="col-sm-9 offset-sm-3">
+				<div class="form-check">
+					<input class="form-check-input" type="checkbox" id="com-users-is-default-method" <?php echo $this->record->default ? 'checked="checked"' : ''; ?> name="default">
+					<label class="form-check-label" for="com-users-is-default-method">
+						<?php echo Text::_('COM_USERS_MFA_EDIT_FIELD_DEFAULT'); ?>
+					</label>
+				</div>
+			</div>
+		</div>
+
+		<?php if (!empty($this->renderOptions['pre_message'])): ?>
+		<div class="com-users-method-edit-pre-message text-muted mt-4 mb-3">
+			<?php echo $this->renderOptions['pre_message'] ?>
+		</div>
+		<?php endif; ?>
+
+		<?php if (!empty($this->renderOptions['tabular_data'])): ?>
+		<div class="com-users-method-edit-tabular-container">
+			<?php if (!empty($this->renderOptions['table_heading'])): ?>
+			<h4 class="h3 border-bottom mb-3">
+				<?php echo $this->renderOptions['table_heading'] ?>
+			</h4>
+			<?php endif; ?>
+			<table class="table table-striped">
+				<tbody>
+				<?php foreach ($this->renderOptions['tabular_data'] as $cell1 => $cell2): ?>
+				<tr>
+					<td>
+						<?php echo $cell1 ?>
+					</td>
+					<td>
+						<?php echo $cell2 ?>
+					</td>
+				</tr>
+				<?php endforeach; ?>
+				</tbody>
+			</table>
+		</div>
+		<?php endif; ?>
+
+		<?php if ($this->renderOptions['field_type'] == 'custom'): ?>
+			<?php echo $this->renderOptions['html']; ?>
+		<?php else: ?>
+		<div class="row mb-3">
+			<?php if ($this->renderOptions['label']): ?>
+			<label class="col-sm-3 col-form-label hasTooltip" for="com-users-method-edit-code">
```

Comment:
```suggestion
			<label class="col-sm-3 col-form-label hasTooltip" for="com-users-method-code">
```

the value of for must match the id